bool value  # Boolean value of the parameter
---
